docs/menu.yml

```diff
@@ -49,14 +49,22 @@ catalog:
               path: /en/setup/configuration/override-settings
         - name: Examples
           catalog:
-            - name: Forward Agent Data to OAP Server
-              path: /en/setup/examples/forward-agent-data-to-oap-server/readme
-            - name: Transmit Log to Kafka
-              path: /en/setup/examples/transmit-log-to-kafka/readme
-            - name: GRPC Load Balance Client
-              path: /en/setup/examples/grpc-load-balance-client/readme
-            - name: High Throughput Queue
-              path: /en/setup/examples/high-throughput-queue/readme
+            - name: Deploy
+              catalog:
+                - name: Deploy on Linux
```

Comment:
BTW, do we have Windows booting script?

Reply:
We haven't for now, I could create it for windows.

Reply:
If you want to add it, update this menu with is, I am totally OK without it for now.
For load balancing, I am a fan of inside k8s 😄

Reply:
been add the `startup.cmd` script for the windows users.